front/app/components/admin/PostManager/InputManager.tsx

```diff
@@ -300,15 +291,8 @@ const InputManager = ({
         </LeftColumn>
         <MiddleColumnTop>
           <IdeasCount
-            feedbackNeeded={
-              feedbackNeeded === true ? feedbackNeeded : undefined
```

Comment:
onChangeFeedbackFilter already deals with this behavior (setting feedback_needed to undefined when the filter goes off), and so we access the same values via the queryParameters.